scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1805,22 +1816,26 @@
   (let [{:keys [pod-condition-unschedulable-seconds
                 synthetic-pod-condition-unschedulable-seconds]}
         (config/kubernetes)
+        {:keys [kubernetes-scheduler-pod-condition-unschedulable-seconds]}
+        (config/kubernetes-scheduler)
         unschedulable-seconds
         (if (some-> pod-name synthetic-pod?)
           synthetic-pod-condition-unschedulable-seconds
-          pod-condition-unschedulable-seconds)]
+          (if (some-> pod-name kubernetes-scheduler-pod?)
+            kubernetes-scheduler-pod-condition-unschedulable-seconds
+            pod-condition-unschedulable-seconds))]
```

Comment:
Yes during testing I observed that when a job uses the k8s scheduler and triggers a node scale-up, the `pod-condition-unschedulable-seconds` would kill the instance after 2 minutes. This is because this setting originally assumed any real job pod would be placed on a node and not be used for autoscaling. So `kubernetes-scheduler-pod-condition-unschedulable-seconds` in the new `kubernetes-scheduler` config allows us to configure how long we allow a real job pod to remain unscheduled before killing it. Currently I set it to the same value we were using for synthetic pods. 

One thing I just noticed is the key for that new config is wrong. The `pod-unschedulable` must now just be waiting infinite time. Will fix shortly.